Source/SubtitleProvider.h

```diff
@@ -213,6 +213,42 @@ class SubtitleProvider :
         SetSubtitleDelay(newDelay);
     }
 
+    int parseInt(std::wstring const& str)
```

Comment:
I just noticed: I moved those methods from SubtitleProvider.h to SubtitleProviderSsaAss.h on winui branch, because the methods are not relevant for other types of subs. I think you accidentially added them back. Also your fix on the hex values is probably not used. You shoud apply it at SubtitleProviderSsaAss.h and remove the methods here.